src/app/components/map/map.component.ts

```diff
@@ -48,20 +49,37 @@ export class MapComponent implements OnInit {
     // this.cluster = this.route.snapshot.queryParamMap.get('mapType');
     this.environment = environment;
     this.map = new L.Map('map', {
-     center: [40, -80],
-     zoom: 3
+      center: [40, -80],
+      zoom: 3,
+      maxZoom: 19
     });
 
-    const baseOSM = L.tileLayer('https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', {
-      maxZoom: 19,
-      attribution: '&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors'
+    this.geoDataService.tileServers.subscribe((next: Array<TileServer>) => {
+      if (next) {
+        this.tileServers = next;
+
+        next.forEach((ts) => {
+          if (!this.tileServerLayers[ts.id]) {
+            this.tileServerLayers[ts.id] = this.tileServerToLayer(ts);
+          }
+
+          this.tileServerLayers[ts.id].setZIndex(ts.uiOptions.zIndex);
```

Comment:
I feel like we should match the behavior of QGIS here (https://docs.qgis.org/3.16/en/docs/user_manual/introduction/general_tools.html#layer-order-panel) where the things higher on the list are "on top" of things in the map.